sys/include/fmt.h

```diff
@@ -214,10 +214,10 @@ size_t fmt_str(char *out, const char *str);
  *
  * Will convert up to @p n digits. Stops at any non-digit or '\0' character.
  *
- * @param[out]  str  Pointer to string to read from
+ * @param[in]   str  Pointer to string to read from
  * @param[in]   n    Maximum nr of characters to consider
```

Comment:
(or if you go for American English you can also go "#", but I think that's confusing ;-))